src/test/java/uk/ac/ebi/atlas/configuration/AbstractContainerBaseTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.ac.ebi.atlas.configuration;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.flywaydb.core.Flyway;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+import javax.sql.DataSource;
+
+abstract class AbstractContainerBaseTest {
+    static final PostgreSQLContainer TEST_DATABASE;
+    static final String DOCKER_IMAGE_VERSION = "postgres:10-alpine";
+    static final String FLY_WAY_PATH = "filesystem:./src/test/resources/schemas/flyway/scxa";
```

Comment:
Notice that these strings are used only once. I think it’s better in that case to inline them, so there are no indirections when reading the code.

Reply:
Sure